Slim/Plugin/xPL/Plugin.pm

```diff
@@ -14,6 +14,8 @@ package Slim::Plugin::xPL::Plugin;
 # xPL Protocol Support Plugin for Logitech Media Server
 # http://www.xplproject.org.uk/
 
+# $Id: Plugin.pm 10841 2006-12-03 16:57:58Z adrian $
```

Comment:
Where are these two lines coming from? Please remove them - otherwise looking good!

Reply:
In the original distro version where I started, looks to be slightly out of sync, now fixed